Sphinxsetup.sh

```diff
@@ -47,6 +47,9 @@ python3 -m pip install --user --upgrade git+https://github.com/ArduPilot/sphinx_
 python3 -m pip install --user --upgrade git+https://github.com/sphinx-contrib/youtube.git
 
 # and a vimeo plugin:
-python3 -m pip install --user --upgrade git+https://github.com/ArduPilot/sphinxcontrib.vimeo.git
+# python3 -m pip install --user --upgrade git+https://github.com/ArduPilot/sphinxcontrib.vimeo.git
+# we previously used this plugin, but it is now included in the youtube plugin
+# remove if installed (will warn if not installed)
+python3 -m pip uninstall sphinxcontrib.vimeo
```

Comment:
maybe a test here with python -m pip show to see if it is installed or not first

Reply:
Fixed. Should not give any output if the package is not present.